packages/common/src/utils/filters.ts

```diff
@@ -329,7 +329,7 @@ export const createDashboardFilterRuleFromField = ({
             disabled: !isTemporary,
             label: undefined,
         },
-        value === null ? undefined : [value],
+        value ? [value] : null, // When `null`, don't set default value if no value is provided
```

Comment:
If value is `falsy`, don't set the default value that is handled by `getFilterRuleWithDefaultValue`

This works OK when: 
`filtering dashboard to...` and when `creating a filter`

Reply:
so in this case if value is `null` then we don't need to include in the values array. code is a bit confusing but comment makes it clear. thanks